.circleci/config.yml

```diff
@@ -433,8 +438,9 @@ workflows:
           name: build_ubuntu_noble
           dist: ubuntu
           release: noble
+          rclass: large
           extra_conf: --enable-asan --enable-ubsan --enable-workspace-emulator
-          make_target: check -j2
+          make_target: check
```

Comment:
Is this a typo? The large instance have 4 vCPU (instead of 2 for medium), should we not use `-j4` here?

Reply:
`-j4` is the default, so we no longer need a `-j2` override.

Reply:
Aha, nvm then.